packages/components/simple-table/src/simple-table.story.js

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, number, boolean, select } from '@storybook/addon-knobs';
+import { Value } from 'react-value';
+import SimpleTable from './simple-table';
+
+const items = [
+  {
+    key: 'camilo-jimenez',
```

Comment:
Maybe we should come up with other names 😅

Reply:
Yeah, I was going to change this at some point 😆 For now I just took the same set of data that we had